README.md

```diff
@@ -40,14 +40,12 @@ Add the following to your `Podfile`:
 2. Add Backtrace multiplatform binary framework bundle to your project using the method that best fits your workflow:
     * Drag & drop `.framework` or `.xcframework` from Finder into Xcode's Project Navigator and check the Target Membership setting
     * Using Swift Package Manager's `binaryTarget` flag
-    * Using CocoaPods's `vendored_frameworks` flag
+    * Using CocoaPods's `vendored_frameworks` flag <br><br>
```

Comment:
If we want to break line maybe we should put it in a new line?

Reply:
it looked off with a new line